keyboard/ergodox_ez/Makefile

```diff
@@ -88,19 +88,19 @@ OPT_DEFS += -DBOOTLOADER_SIZE=512
 
 
 # Build Options
-#   comment out to disable the options.
+#   Set to yes to enable by default, or make MIDI_ENABLE=yes or
+#   make SLEEP_LED_ENABLE=no to override for a given make invocation
```

Comment:
Not sure i follow this comment; care to rephrase maybe? Do you mean to say commenting out no longer works basically, but I need to specify yes/no? What happens if I do comment it out?